details/CLUSTER_DEFINITION.md

```diff
@@ -43,14 +43,92 @@ __Note:__ upon cluster creation, some of the attributes shown below MUST NOT be
                 "beta.kubernetes.io/arch": "amd64",
                 "beta.kubernetes.io/os": "linux",
                 "ip": "10.3.62.2",
-                "kubernetes.io/hostname": "worker-2.x882ofna.k8s.gigantic.io",
+                "kubernetes.io/hostname": "worker-2.wqtlq.example.com",
                 "nodetype": "hiram"
             }
         }
     ]
 }
 ```
 
+## Example 2: Cluster running on AWS EC2 {#example-aws}
+
+This example here shows a cluster definition in JSON format, as it may be submitted for cluster creation, for a cluster running on AWS:
+
+```json
+{
+    "name": "A Cluster on AWS",
+    "owner": "acme",
+    "aws": {
+      "resource_tags": {
+        "costcenter": "166380"
+      }
+    },
+    "workers": [
+        {
+            "aws": {
+              "instance_type": "m3.large"
+            },
+            "labels": {
+                "nodetype": "standard"
+            }
+        },
+        {
+            "aws": {
+              "instance_type": "r3.xlarge"
+            },
+            "labels": {
+                "nodetype": "big-ram"
+            }
+        }
+    ]
+}
```

Comment:
I like that a lot better. Thanks. The object below is still the older version though.